src/Microsoft.ServiceFabric.Actors/Runtime/ActorTypeInformation.cs

```diff
@@ -158,10 +158,10 @@ public static ActorTypeInformation Get(Type actorType)
                     "actorType");
             }
 
-            // get actor event interfaces
+            // Gets actor event interfaces.
```

Comment:
This is a code comment telling the follow of the code, this change is not needed.